crates/ty_python_semantic/resources/mdtest/type_qualifiers/final.md

```diff
@@ -88,9 +88,7 @@ class C:
         self.FINAL_C: Final[int] = 1
         self.FINAL_D: Final = 1
         self.FINAL_E: Final
-        # TODO: Should not be an error
-        # error: [invalid-assignment] "Cannot assign to final attribute `FINAL_E` on type `Self@__init__`"
-        self.FINAL_E = 1
+        self.FINAL_E = 1  # OK: Final can be initialized in __init__
```

Comment:
I don't think we need to comment on every non-error, unless there's something notable or unusual about it.
```suggestion
        self.FINAL_E = 1
```